bedrock/mozorg/redirects.py

```diff
@@ -629,4 +629,7 @@ def decider(request, **kwargs):
 
     # Bug 1313023
     redirect(r'^story/?$', 'https://donate.mozilla.org/?source=story_redirect'),
+
+    # Bug 682619
+    redirect(r'^support/thunderbird/?$', 'https://support.mozilla.org/en-US/products/thunderbird'),
```

Comment:
of course. Stupid me. Should have noticed the redirect above :disappointed: 

I was worried about the locale. thank you :smiley: